src/test/java/org/gridsuite/securityanalysis/server/SecurityAnalysisControllerTest.java

```diff
@@ -673,11 +674,8 @@ void runTestWithError() throws Exception {
         UUID resultUuid = mapper.readValue(resultAsString, UUID.class);
         assertEquals(RESULT_UUID, resultUuid);
 
-        // Message stopped has been sent
-        Message<byte[]> cancelMessage = output.receive(TIMEOUT, "sa.failed");
-        assertEquals(RESULT_UUID.toString(), cancelMessage.getHeaders().get("resultUuid"));
-        assertEquals("me", cancelMessage.getHeaders().get("receiver"));
-        assertEquals(getFailedMessage(COMPUTATION_TYPE) + " : " + ERROR_MESSAGE, cancelMessage.getHeaders().get("message"));
+        // No result message
+        assertNull(output.receive(TIMEOUT, "sa.result"));
```

Comment:
I think the equivalent test is to check if there is a .ComputationException with the corresponding message?

Reply:
can we check the run.dlx?

Reply:
no sadly it doesn't work... the default test binder setup by springcloudstream doesn't behave the same, then I couldn't test like that